Description: Disable version checks in GUI
 Since we handle updates in Debian ourselves, don't nag the user
 when an upstream update is available.
Author: Jeff Licquia <licquia@debian.org>
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/731773
Last-Update: 2014-03-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/gui/src/MainWindow.cpp
+++ b/src/gui/src/MainWindow.cpp
@@ -127,7 +127,7 @@
 
 	showNormal();
 
-	m_VersionChecker.checkLatest();
+	// m_VersionChecker.checkLatest();
 
 	if (appConfig().processMode() == Desktop) {
 		startSynergy();
